### DISCUSSION TITLE
how to get login account by ProjectCommit class

### DISCUSSION THREAD

**Discussion by xgz1981** [python-gitlab/python-gitlab#1899] [Q&A]
## Description of the problem, including code/CLI snippet
Getting project's commit, and the result is ProjectCommit list. I get the attribute committer_name from ProjectCommit. The account is not the same as the user login account. How to get user login account from ProjectCommit?

The gitlab already used ldap to authenticate user, so the user name is like 'xxx.yyy'.  The user maybe set the auhtor and committer another name, when he commit the code.

## Expected Behavior
'committer_name': 'mou.zhang'

## Actual Behavior
'committer_name': 'Alex'

## Specifications

  - python-gitlab version: 3.1.1
  - API version you are using (v3/v4): v4
  - Gitlab server version (or gitlab.com): GitLab Community Edition [11.9.1]

**Comment by JohnVillalovos** [comment]
`python-gitlab` only returns back what is returned from the GitLab server.

You may want to look at the API documentation at: https://docs.gitlab.com/ee/api/commits.html

My guess is that you will need to figure out how to map the `committer_email` attribute to the GitLab user.
